ui/e2e/src/utils/mockTimeSeries.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { PrometheusDatasource } from '@perses-dev/prometheus-plugin';
+
+type PrometheusClient = ReturnType<typeof PrometheusDatasource['createClient']>;
+type RangeQuery = Awaited<ReturnType<PrometheusClient['rangeQuery']>>;
+
+type MockStableValueResultConfig = {
+  metric: Record<string, string>;
+  value: string;
+};
+
+type MockStableValueConfig = {
+  metrics: MockStableValueResultConfig[];
+  startTimeMs: number;
+  endTimeMs: number;
+  count?: number;
+};
+
+export function mockTimeSeriesResponseWithStableValue({
```

Comment:
This is a very simple flat line as an example for how to mock responses. We'll probably want to have more complex examples in the future, but I wanted to keep this PR as a simple starting point.